extensions/sync-data-transfer/sync-data-transfer-api/src/main/java/org/eclipse/dataspaceconnector/transfer/sync/api/SyncDataTransferValidationApiExtension.java

```diff
@@ -53,11 +54,11 @@ public String name() {
      */
     @Override
     public void initialize(ServiceExtensionContext context) {
-        var validationRules = Arrays.asList(new ContractValidationRule(contractNegotiationStore), new ExpirationDateValidationRule());
```

Comment:
Is it correct, that the `ContractValidationRule` was removed from the list? I couldn't find any other usages of this class.